### DISCUSSION TITLE
Epoch 1.16 Test - Rewards dropped drastically

### DISCUSSION THREAD

**Discussion by leeyemi** [tearust/teaproject#20] [Q&A] [answered]
Just noticed the rewards projection calculation show huge variance overnight
When it was at around block height 4000 the projection rewards around 19000
Then this morning at block height 10630 the reward projection shows 13000

What cause the rewards dropped drastically?

Thanks,
Yemi

**Comment by kevingzhang** [comment, marked as answer]
Yes, the epoch 1.14 did not add the age into the performance calculation. Your CML get the peak performance since it is planted. 
epoch 1.16 has the age. You can see "current/peak performance" in the UI. 
Please take a look at the epoch 1.16 What's new https://github.com/tearust/teaproject/discussions/14

**Comment by kevingzhang** [comment]
there are many factors change the reward, age is one of them. others are:
- how many CML are mining?
- staking
- peak performance

**Reply by leeyemi** [reply]
What I mean in the same epoch 1.16 
at block height 4000 the rewards projection is 19k
then block height 10600 the rewards projection is 13000

I understand if they are additional people stake on my farm but this is same scenario (number of staker, number or mining).

How many CML not sure cause all in USD on leader board. However the currency exchange may play the significant different.

**Reply by kevingzhang** [reply]
at the beginning, for example block 4000, there are very few CML planted. Your CML are among the few planted CML. So the revenue is very high due to lack of competition. At block height 10600, there might be much more CML planted and compete with you. Also, there might be other stakers on your CML share your revenue. for example: 
![image](https://user-images.githubusercontent.com/1761809/129670896-43cccd55-4749-49b2-b861-c278aa3473f8.png)

You can see how the stakers affect your revenue.

Another reason would be the exchange rate. We use COFFEE instead of USD now. The exchange rate change will significant affect the number. But in epoch 2.0, you can change what currency you use, so you can use TEA instead of COFFEE.
